z3c/dependencychecker/modules.py

```diff
@@ -115,11 +116,31 @@ def _process_ast_node(self, node):
                     file_path=self.path,
                     is_test=self.testing,
                 )
+        elif isinstance(node, ast.Try):
+            if not HAS_REPORTED_CONDITIONAL_IMPORTS:
+                self._is_a_conditional_import(node)
 
     @staticmethod
     def _is_relative_import(import_node):
         return import_node.level > 0
 
+    def _is_a_conditional_import(self, node):
+        global HAS_REPORTED_CONDITIONAL_IMPORTS
+        for handler in node.handlers:
+            type_info = getattr(handler, 'type', False)
+            if type_info:
+                exception_id = getattr(type_info, 'id', False)
+                if exception_id == 'ImportError':
+                    print('This distribution has conditional imports')
+                    HAS_REPORTED_CONDITIONAL_IMPORTS = True
```

Comment:
The basic idea was to just report it, right? Just for information?

Is it easy to at a filename+linenumber at this point?